dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -202,7 +204,6 @@ StableValueSpacePtr createNewStable( //
         };
         wbs.data.putRemoteExternal(dtfile_id, loc);
     }
-    delegator.addDTFile(dtfile_id, dtfile->getBytesOnDisk(), store_path);
```

Comment:
Delegator is mainly introduced for multi disks management, so it's not necessary for S3.
And to use Delegator, we must reload all DTFiles for the current store, but it's hard to know it for S3 because the store may reference other store's DTFile. So we decide to not use delegator anymore on remote mode now.